src/dotnet/APIView/APIViewWeb/Models/PullRequestModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace APIViewWeb.Models
+{
+    public class PullRequestModel
```

Comment:
Why do we need to store the list of pull requests if the build pipeline would call into API view for every commit anyway?

Reply:
We do not want to trigger these checks again if someone decides to trigger pipeline for same SHA again and again. We also have plans to generate and retain review for architects to approve as alternate option to manual review. These PR API reviews will be marked as closed later if PR is closed. These are part of changes around SDK automation and APIView integration.